cmake/ValhallaSourceGroups.cmake

```diff
@@ -1,15 +1,17 @@
 # create source groups for IDEs such as Visual Studio & XCode
 
 function(create_source_groups prefix)
-  foreach(file ${ARGN})
-    get_filename_component(file "${file}" ABSOLUTE)
-    string(FIND "${file}" "${PROJECT_BINARY_DIR}/" pos)
-    if(pos EQUAL 0)
-      source_group(TREE "${PROJECT_BINARY_DIR}/" PREFIX "Generated Files" FILES "${file}")
-    else()
-      source_group(TREE "${PROJECT_SOURCE_DIR}/" PREFIX "${prefix}" FILES "${file}")
-    endif()
-  endforeach()
+  if (NOT WIN32)
```

Comment:
I did _not_ have a good CI day.. was fighting it longer than I took for the whole PR..  for absolutely no reason win started misbehaving and wouldn't set the source groups bcs of path issues. 

is anyone even using this in their IDE? I never did, vs code doesn't know what to do with it. I think it's for xcode & visual studio? could remove it completely?